src/components/Popup.js

```diff
@@ -1,24 +1,29 @@
 export default class PopUp {
   constructor({ popupSelector }) {
     this._popupElement = document.querySelector(popupSelector);
+    this._handleEscClose = this._handleEscClose.bind(this);
   }
 
   open() {
     this._popupElement.classList.add("modal_open");
     document.addEventListener("keydown", this._handleEscClose);
+
   }
 
   close() {
     this._popupElement.classList.remove("modal_open");
     document.removeEventListener("keydown", this._handleEscClose);
   }
 
-  _handleEscClose(evt) {
+  _handleEscClose (evt) {
     if (evt.key === "Escape") {
       this.close();
-      console.log("escape key");
+
     }
+
   }
+
+
   _cardClickClose(evt) {
```

Comment:
`_cardClickClose`

It's not about `cards`


It's about `overlay`


`_handleOverlay`

Reply:
So does the method name just need to be renamed?